File: 0705-design-hashset/0705-design-hashset.py
class MyHashSet:

    def __init__(self):
        self.arr = [0]*1000001

    def add(self, key: int) -> None:
        self.arr[key] = 1

    def remove(self, key: int) -> None:
        self.arr[key] = 0

    def contains(self, key: int) -> bool:
        return self.arr[key]


# Your MyHashSet object will be instantiated and called as such:
# obj = MyHashSet()
# obj.add(key)
# obj.remove(key)
# param_3 = obj.contains(key)

File: 0705-design-hashset/0705-design-hashset.kt
class MyHashSet() {
    val arr = Array<Boolean>(1_000_001) {false}
    fun add(key: Int) {
        arr[key] = true
    }

    fun remove(key: Int) {
        arr[key] = false
    }

    fun contains(key: Int): Boolean {
        return arr[key]
    }

}

/**
 * Your MyHashSet object will be instantiated and called as such:
 * var obj = MyHashSet()
 * obj.add(key)
 * obj.remove(key)
 * var param_3 = obj.contains(key)
 */